apps/taquito-test-dapp/src/config.ts

```diff
@@ -7,6 +7,7 @@ const rpcUrls: Record<SupportedNetworks, string> = {
   [NetworkTypeBeacon.MAINNET]: "https://mainnet.tezos.ecadinfra.com",
   [NetworkTypeBeacon.GHOSTNET]: "https://ghostnet.tezos.ecadinfra.com/",
   [NetworkTypeBeacon.PARISNET]: "https://rpc.pariscnet.teztnets.com/",
+    // [NetworkTypeWc.QUEBECNET]: "http://ecad-tezos-quebecnet-rolling-1.i.ecadinfra.com/",
```

Comment:
We can't support quebecnet in test-dapp yet because beacon hasn't add quebec support and we rely on their `networkType`.  Hoping by the time of official quebec release beacon would have add the support

Reply:
thanks